appinventor/components-ios/src/StringUtil.swift

```diff
@@ -74,4 +74,122 @@ public func toString(_ object: Any?) -> String {
   @objc static func reverseString(_ string: String) -> String {
     return String(string.reversed())
   }
+    
+    /// Splits text into parts based on a separator
+    public static func split(_ text: String, at: String) -> [String] {
+        var parts = text.components(separatedBy: at)
+        if at.isEmpty && parts.first == "" {
+            parts.removeFirst()
+        }
+        return parts
+    }
+
+    
+    // MARK: - Mapping Methods
+    
+    /// Replaces text using dictionary order
+    @objc public static func replaceAllMappingsDictionary(
+        _ text: String,
+        _ mappings: YailDictionary
+    ) -> String {
+        return replaceAllMappings(text, mappings: mappings, order: mappingOrderDictionary)
+    }
+    
+    /// Replaces text using longest string first order
+    @objc public static func replaceAllMappingsLongestString(
+        _ text: String,
+        _ mappings: YailDictionary
+    ) -> String {
+        return replaceAllMappings(text, mappings: mappings, order: mappingOrderLongestStringFirst)
+    }
+    
+    /// Replaces text using earliest occurrence order
+    @objc public static func replaceAllMappingsEarliestOccurrence(
+        _ text: String,
+        _ mappings: YailDictionary
+    ) -> String {
+        return replaceAllMappings(text, mappings: mappings, order: mappingOrderEarliestOccurrence)
+    }
+    
+    /// Main replacement method with custom ordering
+    public static func replaceAllMappings(
+        _ text: String,
+        mappings: YailDictionary,
+        order: MappingOrder
```

Comment:
Jenkins is complaining `MappingOrder` is not defined. Did you forget to add some files and changes to the project file?

Reply:
Looks like a merge kerfuffle as `MappingOrder` was committed at 5b14f7690c6b2fd4dd72cb0a326726127bac02e7. will add back in and test